core/pkg/hostsensorutils/hostsensor.yaml

```diff
@@ -52,12 +53,19 @@ spec:
         volumeMounts:
         - mountPath: /host_fs
           name: host-filesystem
-        readinessProbe:
+        startupProbe:
           httpGet:
-            path: /kernelVersion
+            path: /readyz
             port: 7888
-          initialDelaySeconds: 1
-          periodSeconds: 1
+          initialDelaySeconds: 10
+          failureThreshold: 10
+          periodSeconds: 10
```

Comment:
I think you can still use 1s here because we'll stop probing once this succeeds

Reply:
ok